handler/aggregate/loader_test.go

```diff
@@ -130,6 +150,57 @@ var _ = Describe("type Loader", func() {
 				))
 			})
 
+			It("returns an instance with the snapshot base root when snapshot is present", func() {
+				ssBase := &AggregateRoot{
+					AppliedEvents: []dogma.Message{
+						MessageE1,
+						MessageE2,
+						MessageE3,
+					},
+				}
+
+				var packet marshalkit.Packet
+				var err error
+				packet, err = Marshaler.Marshal(ssBase)
```

Comment:
```suggestion
				packet, err := Marshaler.Marshal(ssBase)
```

Pedantic Go developer shit.